.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+---
+steps:
+  - type: "waiter"
```

Comment:
This is a line that's often there, but apparently  not needed (from my latest pairing with lang), not in any way a problem though FYI.